qemu/tests/blockdev_commit_with_ioerror.py

```diff
@@ -0,0 +1,94 @@
+from aexpect import ShellTimeoutError
+
+from virttest import utils_numeric
+from virttest import utils_disk
+
+from provider import backup_utils
+from provider.blockdev_commit_base import BlockDevCommitTest
+
+
+class BlockdevCommitWithIoerror(BlockDevCommitTest):
+
+    def generate_tempfile(self, root_dir, filename, size="1000M", timeout=20):
+        """Generate temp data file in guest"""
+        self.session = self.main_vm.wait_for_login()
+        file_path = "%s/%s" % (root_dir, filename)
+        count = int(
+            utils_numeric.normalize_data_size(
+                size,
+                order_magnitude="M",
+                factor=1024))
+        dd_cmd = self.main_vm.params.get(
+            "dd_cmd", "dd if=/dev/urandom of=%s bs=1M count=%s oflag=direct")
+        mk_file_cmd = dd_cmd % (file_path, count)
+        try:
+            self.session.cmd(mk_file_cmd, timeout=timeout)
+        except ShellTimeoutError as e:
+            if self.params["error_msg"] not in str(e):
+                self.test.fail(str(e))
```

Comment:
The error_msg check is not required here, for we defined ShellTimeoutError with that error msg, i.e. once ShellTimeoutError is caught, we always get that message

Reply:
As per above, error_msg can be removed from cfg